dev/Dockerfile

```diff
@@ -9,7 +9,9 @@ RUN apt-get install -y --no-install-recommends \
         libsasl2-modules
 ENV SETUPTOOLS_USE_DISTUTILS=stdlib
 
-COPY ../  ${AIRFLOW_HOME}/astro_sdk/
+COPY ../src  ${AIRFLOW_HOME}/astro_sdk/
+COPY ../pyproject.toml  ${AIRFLOW_HOME}/astro_sdk/
+COPY ../README.md  ${AIRFLOW_HOME}/astro_sdk/
```

Comment:
without copying README getting an error. Most likely because pyproject.toml refer it.

```
 50.69   Traceback (most recent call last):
#0 50.69     File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
#0 50.69       main()
#0 50.69     File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
#0 50.69       json_out['return_val'] = hook(**hook_input['kwargs'])
#0 50.69     File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 144, in get_requires_for_build_editable
#0 50.69       return hook(config_settings)
#0 50.69     File "/tmp/pip-build-env-h_uqtbzk/overlay/lib/python3.9/site-packages/flit_core/buildapi.py", line 23, in get_requires_for_build_wheel
#0 50.69       info = read_flit_config(pyproj_toml)
#0 50.69     File "/tmp/pip-build-env-h_uqtbzk/overlay/lib/python3.9/site-packages/flit_core/config.py", line 70, in read_flit_config
#0 50.69       return prep_toml_config(d, path)
#0 50.69     File "/tmp/pip-build-env-h_uqtbzk/overlay/lib/python3.9/site-packages/flit_core/config.py", line 97, in prep_toml_config
#0 50.69       loaded_cfg = read_pep621_metadata(d['project'], path)
#0 50.69     File "/tmp/pip-build-env-h_uqtbzk/overlay/lib/python3.9/site-packages/flit_core/config.py", line 445, in read_pep621_metadata
#0 50.69       desc_content, mimetype = description_from_file(readme, path.parent)
#0 50.69     File "/tmp/pip-build-env-h_uqtbzk/overlay/lib/python3.9/site-packages/flit_core/config.py", line 277, in description_from_file
#0 50.69       raise ConfigError(
#0 50.69   flit_core.config.ConfigError: Description file README.md does not exist
```